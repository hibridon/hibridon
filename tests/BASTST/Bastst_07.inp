     BASTST       
   7                             ibasty
  F  F  F  T                     logdfl, airyfl, readpt, bastst
    1.0    1.0     0.1           rstart, rendld, spac
  F                              prairy
  1.150      3.00    25.0 15.00  tolai, rincr, rendai, fstfac
   1                             nerg
   5000.0
   16.47000                      xmu
   30.0000                       rcut
  20  20   5   1   0   0   1     jtot1,jtot2,jtotd,jlpar,numin,numax,nud
  48   1                         lscreen, iprint
    4    1                       nnout,niout
    0   1 2 3 
    0
  F  F  F  F  F                  prlogd, prsmat, prt2, t2test, wrsmat
  F  F  F  F  F                  wrpart, prpart, prxsec, wrxsec, wavefl
  F  T  F  F  F                  noprin, chlist, ipos, nucros, photof
  F  F  F  F  F                  flaghf, csflag, flagsu, ihomo, twomol
  F  F                           rsflag, boundc
   1   0   1                     nstate, ipol, npot
   0.000       7.076       27.44           E-3P0, 3P1, 3P2
   183.9      1                       E-1P1, CMIX
  1.0000E-04   9.000      0.4000       13.00       40.00     3 Pi Parameters
  8.0000E-06   13.80      0.4700       18.00       50.00     3 Sig Parameters
  1.0000E-04   9.000      0.4000       13.00       40.00     1 Pi Parameters
  8.0000E-06   13.80      0.4500       18.00       50.00     1 Sig Parameters
  3.5000E-03   6.500      0.4300      4.3000E-03             1 Sigma Morse
   8.100      1.0000E-04  7.5000E-02                         Gaussian Coupling
nofile.dat
