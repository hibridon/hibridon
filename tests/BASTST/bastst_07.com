inp=bastst_07.inp
out=bastst_07.out
run
save=save_07.inp