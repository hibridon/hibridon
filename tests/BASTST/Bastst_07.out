      BASTST                                     
 DUMMY POTENTIAL FOR TESTING PURPOSES            
 ==============================
     *** Parameters:
 JTOT1  =  20        JTOT2  =  20        JTOTD  =   5        JLPAR  =   1
 NERG   =   1        NUMAX  =   0        NUMIN  =   0        NUD    =   1
 LSCREEN=  48        IPRINT =   1
 FSTFAC =  15.00     RINCR  =  1.000     RCUT   =  30.00     RENDAI =  1.000    
 RENDLD =  1.000     RSTART =  1.000     SPAC   = 0.1000     TOLAI  =  1.000    
 XMU    =  16.47    
     *** 1/3-P-AT system parameters:
 NTERM  =   1        NSTATE =   1        IPOL   =   0        NPOT   =   1
 E-3P0  =  0.000     E-3P1  =  7.076     E-3P2  =  27.44     E-1P1  =  183.9    
 DE(3PI)= 1.0000E-04 DE(3SG)= 8.0000E-06 DE(1PI)= 1.0000E-04 DE(1SG)= 8.0000E-06
 RE(3PI)=  9.000     RE(3SG)=  13.80     RE(1PI)=  9.000     RE(1SG)=  13.80    
 BE(3PI)= 0.4000     BE(3SG)= 0.4700     BE(1PI)= 0.4000     BE(1SG)= 0.4500    
 RL(3PI)=  13.00     RL(3SG)=  18.00     RL(1PI)=  13.00     RL(1SG)=  18.00    
 C(3PI) =  40.00     C(3SG) =  50.00     C(1PI) =  40.00     C(1SG) =  50.00    
 CMIX   =  1.000     ALPHG  = 7.5000E-02 RGAUS  =  8.100     AGAUS  = 1.0000E-04
 DEMOR  = 3.5000E-03 REMOR  =  6.500     BEMOR  = 0.4300     DISSMOR= 4.3000E-03
 LAMMIN:    1
 LAMMAX:    4
 MPROJ:     0
     *** Flags:
 AIRYFL= F    BASTST= T    BATCH = F    CHLIST= T    CSFLAG= F    FLAGHF= F
 FLAGSU= F    IHOMO = F    IPOS  = F    LOGDFL= F    NOPRIN= F    NUCROS= F
 PHOTOF= F    PRAIRY= F    PRLOGD= F    PRPART= F    PRSMAT= F    PRT2  = F
 PRXSEC= F    READPT= F    RSFLAG= F    T2TEST= F    TWOMOL= F    WAVEFL= F
 WRPART= F    WRSMAT= F    WRXSEC= F    BOUNDC= F
 ** Energies:     5000.000000
 NOUT:  4; JOUT:  0   1   2   3
 INDOUT:    0   1
 ==============================

 --------------------------------------------------------------------------
           HIBRIDON SCATTERING CODE V 5.00

      AUTHORS: M. ALEXANDER, D. MANOLOPOULOS, H.-J. WERNER, B. FOLLMEG,
               P. DAGDIGIAN
 CONTRIBUTORS: D. LEMOINE, P. VOHRALIK, G. COREY, R. JOHNSON, T. ORLIKOWSKI,
          A. BERNING, A. DEGLI-ESPOSTI, C. RIST, B. POUILLY, J. KLOS, Q. MA,
          G. VAN DER SANDEN, M. YANG, F. DE WEERD, S. GREGURICK, F. LIQUE
 --------------------------------------------------------------------------
           BUILD CONFIGURATION:
CMAKE BUILD DATE: 2023-03-19 at 04:47
CMAKE BUILD TYPE: Debug
CODE GIT REVISION: 09a9b73441fc08cbd5099cd5c70d005971e70187
SYS: Darwin 22.2.0 x86_64
COMPILER: /opt/intel/oneapi/compiler/2022.2.1/mac/bin/intel64/ifort
COMPILE OPTIONS: 
 --------------------------------------------------------------------------
            CURRENT HARDWARE CONFIGURATION:
 
      Model Name: MacBook Pro
      Model Identifier: MacBookPro14,3
      Processor Name: Quad-Core Intel Core i7
      Processor Speed: 2.9 GHz
      Number of Processors: 1
      Total Number of Cores: 4
      L2 Cache (per Core): 256 KB
      L3 Cache: 8 MB
      Hyper-Threading Technology: Enabled
      Memory: 16 GB
 --------------------------------------------------------------------------

 -----------------------------------------------------------------------------
 All publications resulting from use of the integrators included in the
 Hibridon code must include the following reference:

 D. E. Manolopoulos, J. Chem. Phys. 85, 6425 (1986); M. H. Alexander and D. E.
 Manolopoulos, J. Chem. Phys. 80, 2044 (1987).

 All publications involving the determination ofphotodissociation cross sections
 must also include the following reference:

 M. H. Alexander, Comput. Phys. Commun, 75, 87 (1993).

 All publications investigating flux redistribution must include the following references:

 M. H. Alexander, J. Chem. Phys. 95, 8931 (1991); D. E. Manolopoulos and
 M. H. Alexander, J. Chem. Phys.  97, 2527 (1992).

 All publications resulting from use of the Hibridon package must include
 the following reference:

 HIBRIDON is a package of programs for the time-independent quantum treatment
 of inelastic collisions and photodissociation written by M. H. Alexander,
 D. E.  Manolopoulos, H.-J. Werner, and B. Follmeg, with contributions by
 P. F. Vohralik, D. Lemoine, G. Corey, B. Johnson, T. Orlikowski, A. Berning,
 A. Degli-Esposti, C. Rist, P. Dagdigian, B. Pouilly, G. van der Sanden, M. Yang, F. de Weerd, S. Gregurick, J. Klos, and F. Lique
 -----------------------------------------------------------------------------

 ** LABEL:          BASTST                                     
 ** POT NAME:  DUMMY POTENTIAL FOR TESTING PURPOSES            
 ** DATE:      Mar 19 04:50:02 2023

 *** WARNING:  NAIRY .NE. 1 BUT AIRYFL .EQ. .FALSE.
 ** NMAX= 500  NAIRY= 500
 ** TOTAL NUMBER OF ANISOTROPIC TERMS= 2

 **  CC 1/3 ATOM ; NPOT = 1 ** RMU=  16.4700       E=5000.00   JTOT=   20   JLPAR= 1

 OPEN CHANNELS ELIMINATED WHICH ARE CLOSED AT R=   30.00

   N   S   J    L      EINT(CM-1)
   1   1   0   20        0.000
   2   1   1   20        7.076
   3   1   2   18       27.440
   4   1   2   20       27.440
   5   1   2   22       27.440

 ILAM  LAMBDA  ICOL  IROW   I    IV2    VEE
   1      0      1     1     1     1    1.0000000    
   1      0      2     2     2     2    1.0000000    
   1      0      3     3     3     3    1.0000000    
   1      0      4     4     4     4    1.0000000    
   1      0      5     5     5     5    1.0000000    
ILAM=  1 LAMNUM(ILAM) =      5
   2      2      1     3     6     1  -0.16887221    
   2      2      1     4     7     2   0.14154779    
   2      2      1     5     8     3  -0.17733132    
   2      2      2     2     9     4   0.10000000    
   2      2      2     3    10     5  -0.21193294    
   2      2      2     4    11     6   0.12688648E-01
   2      2      2     5    12     7   0.21195147    
   2      2      3     3    13     8   0.92307692E-01
   2      2      3     4    14     9  -0.12848125    
   2      2      4     4    15    10  -0.99284436E-01
   2      2      4     5    16    11  -0.11653970    
   2      2      5     5    17    12   0.10697674    
ILAM=  2 LAMNUM(ILAM) =     12
ILAM=  3 LAMNUM(ILAM) =     12
ILAM=  4 LAMNUM(ILAM) =     12
 ** TOTAL NUMBER OF NONZERO V2 MATRIX ELEMENTS IS    17
 ** BASTST=.TRUE.; TEST OF SUBROUTINE BASIS
 **** END OF CALCULATION ****
      MAXIMUM NUMBER OF CHANNELS USED WAS:     0
      TIMING:  ELAPSED 00:00:00 / CPU 00:00:00 / MP RATIO    0.5 %
      CURRENT DATE:  Mar 19 04:50:02 2023
 ===============================================================================
